Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD [ "python", "./mgbot.py" ]
```

Comment:
Not sure if I need to copy configs over somehow before running this

Reply:
On a second glance, it should probably work, but might make sense to move your configs into env vars in the future.

Reply:
I like the Docker file, been also meaning to do that for a while. I'll pull it locally and test it out at some point. All we need to copy though is the scripts, run pip install requirements but the run command should be different although that's not really required. Right now this is the run command I use "nohup python mgbot.py > botout.log &" but that's so I don't get stuck in the python terminal after I run it and to redirect the logs to a file which might be useful if we're running it in a container.

Reply:
The `requirements.txt` file is copied separately because that allows the requirements step of the image to be cached separately of the actual code. I think the running in the background and outputting logs to files steps can probably be handled by built-in functions like detached mode and the local logging driver.

Reply:
Ok cool, let me test it out locally and I'll get back to you